Sources/StreamChatSwiftUI/ChatChannel/Gallery/ShareButtonView.swift

```diff
@@ -16,8 +17,11 @@ struct ShareButtonView: View {
         Button(action: {
             self.isSharePresented = true
         }, label: {
-            Image(systemName: "square.and.arrow.up")
-                .font(fonts.bodyBold)
+            Image(uiImage: images.share)
+                .renderingMode(.template)
```

Comment:
Since these three modifiers are used a lot, it might make sense to group them in one modifier and apply that one to the image.

Reply:
Done. Needed to extend `Image` here as it could not be a regular custom modifier due to things like `.resizable` only being applicable to `Image` itself. However, I think it still fits best in the `Modifiers` file.